docs/Quick-Start.md

```diff
@@ -11,16 +11,18 @@ aliases:
 ---
 ## How to install
 
-VictoriaMetrics is distributed in two forms:
+VictoriaMetrics is distributed in the following forms:
```

Comment:
I thought that versions of VictoriaMetrics are just Single-node and Cluster and this fact stays unchanged, so VM Cloud was included as an additional method (compared to Docker or k8s) of running Single-node or Cluster version.

Reply:
I see. From my perspective VM cloud is a separate, managed version of VM. And I think it is worth to list it among other versions